static-crawling/jisigin.py

```
@@ -0,0 +1,265 @@
+   "metadata": {},
+   "source": [
+    "### 6. 여러 페이지 크롤링\n",
+    "페이지를 변경하면서 여러 페이지의 데이터를 크롤링합니다.\n",
+    "\n",
+    "- for page_num in range(1, 4): 1페이지부터 3페이지까지 순차적으로 크롤링합니다.\n",
+    "- 각 페이지에서 데이터를 추출하여 data 리스트에 추가하고, 이를 pandas DataFrame으로 변환하여 엑셀 파일로 저장합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
```

Ruff (0.13.1)

Undefined name `null`

(F821)